app/views/course/condition/achievements/edit.html.slim

```diff
@@ -0,0 +1,5 @@
+h2= t('.header')
+
+== render 'form'
```

Comment:
From the official [document](http://www.rubydoc.info/gems/slim/frames), == is "same as the single equal sign (=), but does not go through the escape_html method." They also use `== render 'footer'` in the example. I don't understand how `render` works exactly but if it generates HTML output, then there's no need to escape it by using `=`.